core-services/prow/02_config/openshift-assisted/assisted-grafana/_prowconfig.yaml

```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
```

Comment:
``bugzilla/invalid-bug`` seems redundant here, no?
similar question to the ``jira/invalid-bug`` label, as we don't have the jira plugin enabled for this repo

Reply:
Sure, this was copy-pasted from other repos so I will also change that in another PR